dockers/docker-database/database_config.json.j2

```diff
@@ -94,6 +95,30 @@
             "separator": ":",
             "instance" : "redis"
         }
+{% if DATABASE_TYPE is defined and DATABASE_TYPE == "dpudb" %}
+        ,
+        "DPU_APPL_DB" : {
```

Comment:
maybe i missed some update, so just checking - have we decided to use DPU_APPL_DB or DASH_APPL_DB and etc.

Reply:
Yes, I want to introduce new databases like, `DPU_APPL_DB`, which follows chassis design. I feel it's clearer to keep the underlay and overlay objects in different databases.